### PR TITLE
ref(ts): Convert eventsTable components to typescript

### PR DESCRIPTION
**Pull request by k-fish**
### Summary
Convert eventsTable and eventsTableRow to typescript

### CODE REVIEW

**Review by billyvg — APPROVED** [review]

File: src/sentry/static/sentry/app/components/eventsTable/eventsTable.tsx [hunk 1]
```diff
@@ -4,8 +4,16 @@ import React from 'react';
...
 
-class EventsTable extends React.Component {
+type Props = {
+  events: Event[];
+  tagList: Tag[];
+  orgId: string;
+  projectId: string;
+  groupId: string;
+};
+class EventsTable extends React.Component<Props> {
```
Comment (priscilawebdev) [inline src/sentry/static/sentry/app/components/eventsTable/eventsTable.tsx:16]:
```suggestion
};

class EventsTable extends React.Component<Props> {
```

File: src/sentry/static/sentry/app/components/eventsTable/eventsTableRow.tsx [hunk 2]
```diff
@@ -7,19 +6,20 @@ import DateTime from 'app/components/dateTime';
...
-  };
+type Props = {
+  hasUser?: boolean;
+  orgId: string;
+  groupId: string;
+  projectId: string;
+  event: Event;
+  tagList: Tag[];
+  organization: Organization;
+} & React.HTMLAttributes<HTMLDivElement>;
```
Comment (priscilawebdev) [inline src/sentry/static/sentry/app/components/eventsTable/eventsTableRow.tsx:20]:
```suggestion
type Props = React.HTMLAttributes<HTMLDivElement> & {
  orgId: string;
  groupId: string;
  projectId: string;
  event: Event;
  tagList: Tag[];
  organization: Organization;
   hasUser?: boolean;
};
```

File: src/sentry/static/sentry/app/components/eventsTable/eventsTableRow.tsx [hunk 3]
```diff
@@ -38,8 +38,8 @@ class EventsTableRow extends React.Component {
         {url =>
           url && (
             <small>
-              {crashFileType}: <a href={`${url}?download=1`}>{event.crashFile.name}</a> (
-              <FileSize bytes={event.crashFile.size} />)
+              {crashFileType}: <a href={`${url}?download=1`}>{event.crashFile?.name}</a> (
+              <FileSize bytes={event.crashFile?.size || 0} />)
```
Comment (priscilawebdev) [inline src/sentry/static/sentry/app/components/eventsTable/eventsTableRow.tsx:42]:
```suggestion
              <FileSize bytes={event.crashFile?.size ?? 0} />)
```

**Review by priscilawebdev — APPROVED** [review]
left a few suggestions but it looks good 👍

**Pull request merged**